src/Symfony/Component/AssetMapper/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=8.1",
+        "composer/semver": "^3.0",
```

Comment:
What about making this an optional dep and throw a useful LogicException in the command when used?

Reply:
I thought about that... but we don't use it in a specific command, we run this in the background when using `importmap:require` or `importmap:update`. So in practice, it WILL be needed. And if we just didn't show the warning if this wasn't installed, then nobody would have it installed and it wouldn't do anything. I couldn't find a nice way to avoid it :)